Sources/Auth/AuthProtocolMethod.swift

```diff
@@ -1,34 +1,28 @@
 import Foundation
 import WalletConnectNetworking
 
-enum AuthProtocolMethod: String, ProtocolMethod {
-    case authRequest = "wc_authRequest"
-    case pairingDelete = "wc_pairingDelete"
-    case pairingPing = "wc_pairingPing"
-
-    var method: String {
-        return self.rawValue
-    }
-
-    var requestTag: Int {
-        switch self {
-        case .authRequest:
-            return 3000
-        case .pairingDelete:
-            return 1000
-        case .pairingPing:
-            return 1002
-        }
-    }
-
-    var responseTag: Int {
-        switch self {
-        case .authRequest:
-            return 3001
-        case .pairingDelete:
-            return 1001
-        case .pairingPing:
-            return 1003
-        }
-    }
+struct AuthRequestProtocolMethod: ProtocolMethod {
+    var method: String = "wc_authRequest"
```

Comment:
why var?

Reply:
because xcode has generated it that as vars. Anyway good point, I am changing for lets